### ISSUE TITLE
Android Audio Delayed

### ISSUE THREAD

**Issue by slime73** [love2d/love#1241]
**[Original report](https://bitbucket.org/rude/love/issue/1236) by Miku AuahDark (Bitbucket: [AuahDark](https://bitbucket.org/AuahDark), GitHub: [MikuAuahDark](https://github.com/MikuAuahDark)).**

----------------------------------------

The audio in Android seems have some delay, which makes rhythm-based game in LOVE2D very hard to play. I'm currently making a rhythm-based game and the audio delay introduced in Android is unacceptable for my rhythm-based game.

Tested in Android Jelly Bean and Lollipop, both introduce audio delay. Tested in Windows and there's no audio delay. Tried to test it in Ubuntu under VM and there's audio delay issue too, but since I'm testing in VM, I can't confirm if the audio is also delayed in real Ubuntu installation.

Is there any workaround for this?

**Comment by MikuAuahDark** [comment]
Recent phones have less latency now, but still noticeable. Also see love2d/love-android#144

TL;DR: Patch no longer applies as LOVE Android now uses OpenAL-soft 1.19.1.
